File: scripts/sample_app5/matlab/plotAvgVmUtilization.m
function [] = plotAvgVmUtilization()

    plotGenericLine(2, 8, 'Average VM Utilization of RSU (%)', 'ALL_APPS', 'NorthWest', 0);

    plotGenericLine(3, 8, 'Average VM Utilization of Cloud (%)', 'ALL_APPS', 'NorthWest', 0);
    
end

File: scripts/tutorial3/python/plotAvgVmUtilization.py
from plotGenericLine import plot_generic_line

if __name__ == '__main__':
    print("--- Generating: Average VM Utilization Plots ---")
    
    plot_generic_line(2, 8, 'Average VM Utilization of Edge (%)', 'ALL_APPS', '', 'upper left')
    plot_generic_line(3, 8, 'Average VM Utilization of Cloud (%)', 'ALL_APPS', '', 'upper left')